docs/redirect-guest-to-cas.md

```diff
@@ -0,0 +1,62 @@
+# Redirecting Guests to CAS for Sign-In
+
+An occasionally feature request is for uPortal to only support authenticated users,
+skipping a guest experience all together.
+
+This can be accomplished by wrapping the CAS Authentication Filter around `/Login`.
+
+Add the CAS Authentication Filter to uportal-war/src/main/webapp/WEB-INF/web.xml:
```

Comment:
Would the path be clearer with some highlighting to the path?
Maybe italics `*text*` or a code highlight `` `text` ``?